### PR TITLE
Fix cupy.random.hypergeometric

### PR DESCRIPTION
**Pull request by toslunar**
This PR fixes the nvrtc error:
```
link error: Linking globals named '_Z3minll': symbol multiply defined!
```

See the results of the daily tests on Jenkins.

### CODE REVIEW

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

File: cupy/random/_kernels.py [hunk 1]
```diff
@@ -674,9 +662,9 @@
     long Z;
     double T, W, X, Y;
 
-    mingoodbad = min(good, bad);
+    mingoodbad = good < bad ? good : bad;
     popsize = good + bad;
-    maxgoodbad = max(good, bad);
+    maxgoodbad = good > bad ? good : bad;
     m = min(sample, popsize - sample);
```
[the remark this answers is not in the document]
Comment (YoshikawaMasashi) [reply cupy/random/_kernels.py]: and `d11 = min(min(m, mingoodbad)+1.0, floor(d6+16*d7));`

**Comment by toslunar** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
Jenkins, test this please.

**Pull request merged**